client/src/components/relatedProducts/RelatedProducts.jsx

```diff
@@ -1,34 +1,79 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import ProductCard from './ProductCard.jsx';
+import Modal from './Modal.jsx';
 
 class RelatedProducts extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      relatedProductIds: [],
       relatedProducts: [],
+      showModal: false,
+      comparedProduct: {
+        product: {
+          name: '',
+        },
+        styles: {},
+      },
     };
+    this.onClickStar = this.onClickStar.bind(this);
+    this.onClickCloseModal = this.onClickCloseModal.bind(this);
   }
 
   // get product id from overview
   componentDidMount() {
-    // make api call to related products
+    // make api call to related products; reset state to empty arrays
+    const newArray = [];
```

Comment:
Would suggest renaming this variable to `relatedProducts`

Reply:
done!